index.js

```diff
@@ -51,7 +56,9 @@ class BlockedMonitor {
         const blockedTime = ms - checkInterval;
 
         if (blockedTime > this.threshold) {
-          logger.error({ blockedTime, serviceName },
+          metrics.increment(`blocked`);
+          metrics.timing(`blocked.duration`, blockedTime);
+          logger[this.loggerLevel]({ blockedTime, serviceName },
```

Comment:
this might be a small optimization, but this lib should be lightweight so I'll add the comment anyway : directly assign `this.logFunction` to avoid accessing it dynamically at each iteration ?

Reply:
Done